main.go

```diff
@@ -999,7 +1000,7 @@ func processImportCfg(flags []string) (newImportCfg string, _ error) {
 			// For beforePath="vendor/foo", afterPath and
 			// lpkg.ImportPath can be just "foo".
 			// Don't use obfuscatedImportPath here.
-			beforePath = hashWith(lpkg.GarbleActionID, beforePath)
+			beforePath = hashWith(getSalt(lpkg), beforePath)
```

Comment:
how about just passing the package to hashWith, then keep the "get salt" logic there?